pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardrefs/create.yaml

```diff
@@ -51,7 +51,6 @@ spec:
                   perSeriesAligner: ALIGN_RATE
               unitOverride: "1"
             plotType: "STACKED_BAR"
-          timeshiftDuration: 0s
```

Comment:
Is `0s` similiar as a `reset` ?

Reply:
It's more banal than that, timeshiftDuration must be a minimum of 60 seconds, and cannot be used with STACKED_BAR.  Because we're testing it above, I thought it easier just to drop this invalid value.

(I only realized this when I ran it against real GCP!)